ee/apps/ddp-streamer/src/Client.ts

```diff
@@ -98,7 +98,7 @@ export class Client extends EventEmitter {
 			this.ws.close(WS_ERRORS.CLOSE_PROTOCOL_ERROR, WS_ERRORS_MESSAGES.CLOSE_PROTOCOL_ERROR);
 		});
 
-		this.setMaxListeners(50);
+		this.setMaxListeners(200);
```

Comment:
This fixes another problem which I didn't investigate why it happens. Node complains we put 51 listeners to this client. IDK, should i keep this change here?

Reply:
(I know it's not a "fix" itself but yeah, you get the point)

Reply:
what needs to be invested is: is it correct to have more than 50 streams being listened ?

Reply:
True. There's a task for that investigation. This change is more of a "shut up" but what I don't know is, should we shut up and then investigate? That would make customer "happy" for a while.

IDK. I'll remove this change from here for now.